src/news/moving-on.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Moving on
+description: The origins of PolyMC are plagued by controversy. Let's move on from this.
+date: 2022-02-15
+tags:
+  - multimc
+---
+
+PolyMC emerged out of frustration with the MultiMC project and therefore brought controversy with it.
+In short, the original founders of PolyMC had disagreements with the maintainer of MultiMC and the project's direction.
+These disagreements were turned into a takeover of packaging on Arch Linux by the PolyMC founders, something that, understandably, the [MultiMC maintainers took as sabotage][gh-poly-meta-sabotage].
+After MultiMC [dropped support for third-party packaging][mmc-drops-aur], the seeds for PolyMC were laid.
+
+As a side note, we want to mention that we do not approve the actions of community members that used these controversies as a basis to harass MultiMC developers.
+Please refrain from brigading other communities.
+It only causes harm on both sides.
+
+PolyMC's inception is very controversial by its very nature.
+The founders have decided to sublicense PolyMC under the terms of the [GNU General Public License 3][gpl3] (GPL-3), which made it impossible for MultiMC to take contributions, without re-licensing to GPL-3 as well.
+As a result the MultiMC team have decided to burn bridges with PolyMC.
+This was manifested after access to the metadata service, provided by MultiMC, was [blocked for all PolyMC users][gh-poly-meta].
+All versions prior to 1.0.6 do not work anymore, as they are trying to reach MultiMC servers instead of our own.
+
+In an effort to prepare for our upcoming 1.1.0 release, we wanted to contact the MultiMC maintainer to see, if they want to be credited in our documentation.
+Sadly, our request was dismissed, and one of our team members was banned from the official MultiMC Discord server for being associated with PolyMC.
+
+<div class="center">
+    <img alt="Screenshot of public chat with MultiMC maintainer" src="/img/news/moving-on/discord.png" style="width: 40em">
+</div>
+
+This has sparked many internal discussions both at PolyMC as well as MultiMC.
+We concluded that we will be moving away from our origins as a packaging-friendly fork of MultiMC to it's own thing.
```

Comment:
`it's` should be `its`, but also the end should say `our own thing` since the subject earlier on is `our origins`. Maybe re-frame the sentence?